dappnode/DNP_DAPPMANAGER
# Update brand assets within public folder [Fixes #1566]

## Context

Simple UI fix to swap out image assets using old branding and replace with updated assets

## Approach

- Created replacements for all icon sizes needed for the web manifests, using the existing `packages/admin-ui/src/img/dappnode-logo-only.png` asset
- Removed old image assets 
- Updated references to point to new images, including the addition of a couple more icon sizes

## Test instructions

I was unable to get my local environment working properly, but you should be able to:

1. Serve the admin-ui with this build
2. Open site in Safari on desktop (consider clearing cache first)
3. The tab bar icon and the share sheet icon should be updated displaying the new image assets instead of the old. 
4. If you can also open this on an iPhone, saving it to the home screen from Safari should show the new icon as well.
